ui/src/app/components/resource-card.spec.tsx

```diff
@@ -82,6 +118,25 @@ describe('ResourceCardComponent', () => {
     expect(lastModifiedDate(wrapper)).toBeTruthy();
   });
 
-  // Note: this spec is not testing the Popup menus on resource cards due to an issue using
-  //    PopupTrigger in the test suite.
+  // We can't test the popup menus themselves in this test because PopupTrigger doesn't know how to draw the modal
+  // without a browser to define bounding boxes...
+  it('should always have a clickable popup menu (cohort)', () => {
+    const wrapper = component(ResourceCardWrapper.readonlyCohortCard);
+    expect(wrapper.find('[data-test-id="resource-menu"]').props()['disabled']).toBeUndefined();
```

Comment:
I removed the `disabled` property because we don't want to disable the triggers anymore. It would probably be clearer to word this test as 'disabled not in props'.